src/App.tsx

```diff
@@ -1,9 +1,21 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const { title, description } = movie;
+    const lowCaseTitle = title.toLowerCase();
+    const lowCaseDescription = description.toLowerCase();
+    const lowCaseQuery = query.toLowerCase();
```

Comment:
let's make boolean conditions like `doesTitleMatch / doesDescriptionMatch` instead of those. Will be clearer this way